global/lang/en_GB/general.json

```diff
@@ -19,7 +19,7 @@
     "close": "Close",
     "language.switch": "Switching language!",
     "locked.login": "Log In to use this feature!",
-    "page.generatedIn": "Page generated in $1 seconds.",

```

Comment:
since we show decimals, the closest we'll get to 1 is 0.9999 second(s) or 1.000001 second(s), it's so so unlikely we'll ever use a singular "second" that im not sure it's a great idea to add here

i'm fine with it but thought context was worth it. keep in mind usually the numbers are more around 0.0606